nextflow.config

```diff
@@ -6,13 +6,11 @@ manifest {
     mainScript = 'main.nf'
```

Comment:
@Mxrcon , let's also update the manifest here 

```nextflow
manifest {
   name = 'mtbseq-nf'
   description = 'MTBSeq is a workflow for the MTBSeq for analysis of Mycobacterium tuberculosis, packaged as a Nextflow Workflow for usability and sensible defaults'
   version = '1.0.0'
   author = 'Emilyn Costa Conceição <emilyncosta@gmail.com> ; Abhinav Sharma <abhi18av@outlook.com>;  Davi  Josue Marcon <davijosuemarcon@gmail.com>'
   mainScript = 'main.nf'
}


```

Reply:
yes sure, but if there's no problem for you, I'd like to add the author names by alphabetic order. what do you think about it?